test/blocks/gist/gist.test.js

```diff
@@ -6,10 +7,43 @@ document.body.innerHTML = await readFile({ path: './mocks/body.html' });
 const { default: init } = await import('../../../libs/blocks/gist/gist.js');
 
 describe('adobetv autoblock', () => {
-  it('creates iframe video block', async () => {
+  let createElementStub;
+  let originalCreateElement;
+
+  beforeEach(() => {
+    originalCreateElement = document.createElement;
+
+    createElementStub = sinon.stub(document, 'createElement').callsFake((tagName) => {
+      if (tagName === 'script') {
+        const scriptMock = originalCreateElement.call(document, 'script');
+        Object.defineProperty(scriptMock, 'src', {
+          set(url) {
+            const callbackMatch = url.match(/callback=([^&]+)/);
+            if (callbackMatch) {
```

Comment:
Let's be more explicit about which callback we expect here, you could also match this based on the gistlink you put in
```suggestion
            const gistCb = url.match(/callback=([^&]+)/);
            if (!gistCb) return
```